.github/workflows/main.yaml

```diff
@@ -21,7 +21,7 @@ jobs:
           go-version: ${{ env.go-version }}
 
       - name: Start etcd container 
-        run: docker run -d --network host quay.io/coreos/etcd:v3.4.16
+        run: docker run -d --network host quay.io/coreos/etcd:v3.5.1
```

Comment:
There is a new repository since v3.2.5
[gcr.io/etcd-development/etcd](https://console.cloud.google.com/gcr/images/etcd-development/GLOBAL/etcd)

Reply:
https://github.com/etcd-io/etcd/releases/tag/v3.2.5
> Starting from `v3.2.5`, we are switching to `gcr.io/etcd-development/etcd`.
> `quay.io/coreos/etcd` will be deprecated *in the future*.

Hmm...
They should emphasize their deprecation policy much more.